src/Sylius/Bundle/TaxonomyBundle/Resources/config/doctrine/model/TaxonTranslation.orm.xml

```diff
@@ -23,11 +23,11 @@
             <generator strategy="AUTO" />
         </id>
         <field name="name" column="name" type="string" />
-        <field name="slug" column="slug" type="string" unique="true">
-            <gedmo:slug fields="name" unique="true" />
+        <field name="slug" column="slug" type="string" unique="false">
+            <gedmo:slug fields="name" unique="false" />
         </field>
-        <field name="permalink" column="permalink" type="string" unique="true">
-            <gedmo:slug fields="name" unique="true" >
+        <field name="permalink" column="permalink" type="string" unique="false">
+            <gedmo:slug fields="name" unique="false" >
                 <!--TODO custom handler for translations-->
```

Comment:
I have to look further into this, as I don't really know what the handlers do, will look at it and update this PR with the solution.